src/components/biggive-footer/biggive-footer.scss

```diff
@@ -78,7 +78,7 @@ nav ul li a {
   }
 
   &.icon-small-charity {
-    background-image: url(/assets/images/triangles-combined-arts-white.svg);
+    background-image: url(/assets/images/triangles-combined-small-charity-week-white.svg);
```

Comment:
https://github.com/thebiggive/components/blob/0b6dd5a9a6d3475747bf88ea5b7319cdfb98c150/src/assets/images/triangles-combined-small-charity-week-white.svg

not https://github.com/thebiggive/components/blob/0b6dd5a9a6d3475747bf88ea5b7319cdfb98c150/src/assets/images/triangles-combined-arts-white.svg